tests/test.jsx

```diff
@@ -0,0 +1,10 @@
+import test from 'tape';
```

Comment:
this is the test file? It doesn't include `jsx` therefore I would have it as a `js` file